bazarr/app/database.py

```diff
@@ -32,13 +32,19 @@ class ReconnectPostgresqlDatabase(ReconnectMixin, PostgresqlDatabase):
             (OperationalError, 'server closed the connection unexpectedly'),
         )
 
+    postgres_database = os.getenv("POSTGRES_DATABASE", settings.postgresql.username)
```

Comment:
Shouldn't it be `settings.postgresql.database`?

Reply:
I fixed it already :)